components/kubeflow/pytorch-launcher/component.yaml

```diff
@@ -0,0 +1,50 @@
+name: Kubeflow - Launch PyTorchJob
+description: Kubeflow PyTorchJob launcher
+inputs:
```

Comment:
I see that this component does not have any outputs. It would be great to have some meaningful outputs.